lib/kitchen/provisioner/dsc.rb

```diff
@@ -144,10 +144,9 @@ def powershell_modules
           params = if powershell_module.is_a? Hash
                      keys = powershell_module.keys.reject { |k| k.to_s.downcase! == 'force' }
                      unless keys.any? { |k| k.to_s.downcase! == 'repository' }
```

Comment:
@PrahlM93 Thanks for the PR and raising the issue. Unfortunately the real problem with this section of code is here with the `downcase!` and not the logic underneath.  I put in #24 to fix this.